src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,225 @@
+                            System.out.println("Please choose a valid list entry.");
+                        }
+                        break;
+                    case "event":
+                    case "deadline":
+                    case "todo":
+                        try {
+                            storeTasks(userLists, userLineInput, numOfTasks, userCommand);
+                            numOfTasks++;
+                        } catch (StringIndexOutOfBoundsException e ) {
+                            System.out.println(TEXT_SPACER +
+                                    "Description of your task can't be empty. At least enter something!");
+                        }
+                        break;
+                    case "delete":
+                        try {
+                            deleteEvent(userLists, splitInputs, numOfTasks);
+                            numOfTasks--;
+                        } catch (ArrayIndexOutOfBoundsException | NullPointerException e){
+                            System.out.println(TEXT_SPACER + "Please choose a valid entry to delete.");
+                        }
+                        break;
+                    default:
+                        throw new DukeExceptions(TEXT_SPACER +
+                                "Sorry mate, I don't understand what you are trying to get me to do.");
+                    }
+
+                    saveToFile(userLists, numOfTasks);
+                } catch (DukeExceptions | IOException e) {
+                    System.out.println(e.getMessage());
+                }
+                printLineSpacer();
+
+            }
+        } while(!userLineInput.equals("bye"));
+
+        System.out.println(TEXT_SPACER + "Aight. See you soon mate.");
+        printLineSpacer();
+    }
+
+    private static int getFile(ArrayList userLists, int numOfTasks) {
+        try {
+            File saveFile = new File(path + "/task.txt");
+            saveFile.createNewFile();
+            Scanner savedTasks = new Scanner(saveFile);
+            while(savedTasks.hasNext()) {
+                String[] currentLine = savedTasks.nextLine().split("\\|");
+                String taskType = currentLine[0];
+                String taskStatus = currentLine[1];
+                String taskCommand;
+                switch (taskType) {
+                case "T ":
+                    taskCommand = "todo" + currentLine[2];
+                    storeToDo(userLists, taskCommand, numOfTasks);
+                    break;
+                case "D ":
+                    taskCommand = "deadline" + currentLine[2];
+                    storeDeadline(userLists, taskCommand, numOfTasks);
+                    break;
+                case "E ":
+                    taskCommand = "event" + currentLine[2];
+                    storeEvent(userLists, taskCommand , numOfTasks);
+                    break;
+                }
```

Comment:
Good job here with the [switch-case statement](https://se-education.org/guides/conventions/java/basic.html#layout) formatting! How about the `default` case? Seems it hasn't been handled here.